wenet/bin/train_nst.py

```diff
@@ -0,0 +1,318 @@
+                        type=int,
+                        help='num of subprocess workers for reading')
+    parser.add_argument('--pin_memory',
+                        action='store_true',
+                        default=False,
+                        help='Use pinned memory buffers used for reading')
+    parser.add_argument('--use_amp',
+                        action='store_true',
+                        default=False,
+                        help='Use automatic mixed precision training')
+    parser.add_argument('--fp16_grad_sync',
+                        action='store_true',
+                        default=False,
+                        help='Use fp16 gradient sync for ddp')
+    parser.add_argument('--cmvn', default=None, help='global cmvn file')
+    parser.add_argument('--symbol_table',
+                        required=True,
+                        help='model unit symbol table for training')
+    parser.add_argument('--prefetch',
+                        default=100,
+                        type=int,
+                        help='prefetch number')
+    parser.add_argument('--bpe_model',
+                        default=None,
+                        type=str,
+                        help='bpe model for english part')
+    parser.add_argument('--override_config',
+                        action='append',
+                        default=[],
+                        help="override yaml config")
+    parser.add_argument('--enable_nst',
+                        default=1,
+                        type=int,
+                        help='whether or not use unsupervised data')
+
+    args = parser.parse_args()
+    return args
+
+
+def main():
+    args = get_args()
+    logging.basicConfig(level=logging.DEBUG,
+                        format='%(asctime)s %(levelname)s %(message)s')
+    os.environ['CUDA_VISIBLE_DEVICES'] = str(args.gpu)
+
+    # Set random seed
+    torch.manual_seed(777)
+    with open(args.config, 'r') as fin:
+        configs = yaml.load(fin, Loader=yaml.FullLoader)
+    if len(args.override_config) > 0:
+        configs = override_config(configs, args.override_config)
+
+    distributed = args.world_size > 1
+    if distributed:
+        logging.info('training on multiple gpus, this gpu {}'.format(args.gpu))
+        dist.init_process_group(args.dist_backend,
+                                init_method=args.init_method,
+                                world_size=args.world_size,
+                                rank=args.rank)
+
+    symbol_table = read_symbol_table(args.symbol_table)
+
+    train_supervised_conf = configs['supervised_dataset_conf']
+    train_unsupervised_conf = configs['unsupervised_dataset_conf']
+
+    print("*********train setting**************")
+    print("supervised data list = ", args.train_data_supervised)
+    print("unsupervised data list = ", args.train_data_unsupervised)
+
+    cv_conf = copy.deepcopy(train_supervised_conf)
+    # cv_conf['speed_perturb'] = False
```

Comment:
why use speed_perturb and spec_aug in cv?

Reply:
Hi robin, thanks for pointing out this issue, the speed_perturb, spech_aug as well as shuffle should be set to false for CV.  I traced back my history and found it was a mis-comment during my recent review. In our experiment cluster, it was set to be False. 
<img width="874" alt="eae283e8e3a124bd12bca4d8f53cd6c" src="https://user-images.githubusercontent.com/59667751/206090960-4cba90fb-489e-4666-97a5-b45293ff77b5.png">